### PR TITLE
mulitple -> multiple in docs

### PR DESCRIPTION
**Pull request by callumacrae**
mulletple?

### CODE REVIEW

**Comment by shentao** [comment]
Holy crap! 😹

Thanks, @callumacrae! How do you do btw? ;>

File: gh-pages/index.html [hunk 1]
```diff
@@ -517,7 +517,7 @@
       isDisabled: false,
       isTouched: false,
       value: [],
-      options: ['Select option', 'Disable me!', 'Reset me!', 'mulitple', 'label', 'searchable']
+      options: ['Select option', 'Disable me!', 'Reset me!', 'multiple', 'label', 'searchable']
```
Comment (zivc) [inline gh-pages/index.html]: The exclamation marks here may be emotional triggers to people who hate shouting. I'd advise you remove the exclamation marks or insert trigger warnings.

Comment (callumacrae) [reply gh-pages/index.html]: omg ash stop following me around github commenting on my pull requests 😭

**Comment by callumacrae** [comment]
@shentao i'm good! i'll message you on slack :)

File: gh-pages/index.html (hunk 1 above)
Comment (shentao) [reply gh-pages/index.html]: Pull request, please! 👻

**Pull request merged**